tgui/packages/tgui/interfaces/TurretControl.js

```diff
@@ -0,0 +1,108 @@
+import { Window } from '../layouts';
+import { useBackend, useLocalState } from '../backend';
+import { Button, Section, Flex, Box, Stack } from '../components';
+
+const randInt = (a, b) => {
+  if (a > b) {
+    let temp = a;
+    a = b;
+    b = temp;
+  }
+  return Math.floor(Math.random() * (b - a + 1)) + a;
```

Comment:
```suggestion
  const min = b > a ? a : b;
  const max = b > a ? b : a;
  return Math.floor(Math.random() * (max - min + 1)) + min;
```
Your thing is "fine", I have a *strong* preference for not touching arguments if you can avoid it. I also would have no issue with having this function limited to require the smaller number first and larger second, as you're only using it in this interface so can control its usage (and thus remove the checks).